### ISSUE TITLE
Incorrect slider position while moving the start  and end times

### ISSUE THREAD

**Issue by helenayele** [debrief/debrief#4843]
https://github.com/debrief/debrief/pull/4836#pullrequestreview-386544991

After setting the start period time when I move it, the slider shows the start time at incorrect location.

The slider should always point at the beginning when I change the start period time.
Other video that shows the start time at the middle:

[![Image from Gyazo](https://i.gyazo.com/8df7c169c6fb74fb502f33b934887729.gif)](https://gyazo.com/8df7c169c6fb74fb502f33b934887729)

**Comment by IanMayo** [comment]
Hmm, I think it's working correctly.

As we move the lower slider, the time remains at 122340.  But, this represents changing points on the upper slider.   Well, it does until we the bottom slider goes further right than 122340, in which case the upper slider jumps to the new time.

**Comment by helenayele** [comment]
Yes, clear. When I move the lower slider , it correctly set the start time on the slider and it always holds the beginning of the slider. But I get the issue after I move the time period to the right or left. Can you please check after that?
